### ISSUE TITLE
gatk4 missing openblas

### ISSUE THREAD

**Issue by micknudsen** [bioconda/bioconda-recipes#12944]
The `CreateReadCountPanelOfNormals` tool in GATK4 appears to be missing a dependency. Looking at the log files, there is an error 

`undefined symbol: cblas_dspr `

even though the tool finishes with exit code 0. The output file, however, is corrupt. A solution (which I have confirmed works) has been posted here:

https://gatkforums.broadinstitute.org/gatk/discussion/8810/something-about-create-pon-workflow

Summary: One must install `openblas` and then run `export LD_PRELOAD=/path/to/libopenblas.so`. Is it possible to somehow add this to the `gatk4` recipe?

**Comment by bgruening** [comment]
Yes, you can depend on openblas as runtime dependency you would probably need to write a small shime around gatk4 if you really need to export LD_PRELOAD.

**Comment by micknudsen** [comment]
Thanks for the info. I will look into it and see if I can figure out how to do that.

**Comment by dpryan79** [comment]
I'll close this issue for now, please feel free to reopen it if anything is still unclear.
